.github/workflows/camunda-platform-release.yml

```diff
@@ -689,4 +713,91 @@ jobs:
       dockerImage: ${{ inputs.dryRun && '' || format('camunda/zeebe:{0}', inputs.releaseVersion) }}
     secrets: inherit
 
+  notify-on-success:
+    name: Send Slack notification on success
+    runs-on: ubuntu-latest
+    needs: [release]
```

Comment:
Actually you were right before. Since notify-on-success/failure was moved into camunda-platform-release.yml, we must explicitly declare all jobs above it. that's it 
```
needs: [release, github, zeebe-docker, operate-docker, tasklist-docker, camunda-docker, snyk]
```